internal/generator/vector/conf_test.go

```diff
@@ -170,8 +170,8 @@ source = """
 [transforms.route_container_logs]
 type = "route"
 inputs = ["container_logs"]
-route.app = '!((starts_with!(.kubernetes.namespace_name,"kube")) || (starts_with!(.kubernetes.namespace_name,"openshift")) || (.kubernetes.namespace_name == "default"))'
-route.infra = '(starts_with!(.kubernetes.namespace_name,"kube")) || (starts_with!(.kubernetes.namespace_name,"openshift")) || (.kubernetes.namespace_name == "default")'
+route.app = '!((starts_with!(.kubernetes.namespace_name,"kube-")) || (starts_with!(.kubernetes.namespace_name,"openshift-")) || (.kubernetes.namespace_name == "default") || (.kubernetes.namespace_name == "openshift"))'
```

Comment:
I think we miss "kube" namespace here

Reply:
there is no `kube` namespace described in https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/

Reply:
perhaps we have missed `openshift` namespace in fluentd also?
```
InfraContainerTags            = "kubernetes.var.log.pods.openshift-*_** kubernetes.var.log.pods.default_** kubernetes.var.log.pods.kube-*_**"
```

https://github.com/openshift/cluster-logging-operator/blob/master/internal/generator/fluentd/source/tags.go#L7